src/Agent.Worker/ExecutionContext.cs

```diff
@@ -578,6 +580,24 @@ private void JobServerQueueThrottling_EventReceived(object sender, ThrottlingEve
             if (!_throttlingReported)
             {
                 this.Warning(StringUtil.Loc("ServerTarpit"));
+
+                if (this.Variables.System_TFCollectionUrl != null)
+                {
+                    // Construct a URL to the resource utilization page, to aid the user debug throttling issues
+                    UriBuilder uriBuilder = new UriBuilder(Variables.System_TFCollectionUrl);
+                    NameValueCollection query = HttpUtility.ParseQueryString(uriBuilder.Query);
+                    DateTime endTime = DateTime.UtcNow;
+                    string queryDate = endTime.AddHours(-1).ToString("s") + "," + endTime.ToString("s");
+
+                    uriBuilder.Path += (Variables.System_TFCollectionUrl.EndsWith("/") ? "" : "/") + "_usersSettings/usage";
+                    query["tab"] = "pipelines";
```

Comment:
double check with Matt, see whether this is the tab we want to show.

Reply:
I double checked with Matt yesterday, and he's good with showing the pipelines tab.